src/pipx/util.py

```diff
@@ -171,6 +168,17 @@ def run_subprocess(
     logger.info(f"running {log_cmd_str}")
     # windows cannot take Path objects, only strings
     cmd_str_list = [str(c) for c in cmd]
+    # Make sure to call the binary using its real path. This matters especially
+    # on Windows when using the packaged app (Microsoft Store) version of
+    # Python, which uses path redirection for sandboxing. If the path to the
+    # executable is redirected, the executable can get confused as to which
+    # directory it's being run from, leading to problems.
+    # See https://github.com/pypa/pipx/issues/1164
+    # Conversely, if the binary is a symlink, then we should NOT use the real
+    # path, as Python expects to receive the symlink in argv[0] so that it can
+    # locate the venv.
```

Comment:
Align this to 120 characheters please https://github.com/pypa/pipx/blob/main/pyproject.toml#L57

Reply:
Done.